pkgs/by-name/ma/maltego/package.nix

```
@@ -0,0 +1,75 @@
+{ lib
+, stdenv
+, fetchzip
+, jre
+, giflib
+, gawk
+, makeBinaryWrapper
+, icoutils
+, copyDesktopItems
+, makeDesktopItem
+}:
+
+stdenv.mkDerivation rec {
+  pname = "maltego";
+  version = "4.6.0";
+  src = fetchzip {
+    url = "https://downloads.maltego.com/maltego-v4/linux/Maltego.v${version}.linux.zip";
+    hash = "sha256-q+1RYToZtBxAIDSiUWf3i/3GBBDwh6NWteHiK4VM1HY=";
+  };
+
+  postPatch = ''
+      substituteInPlace bin/maltego \
+            --replace /usr/bin/awk ${gawk}/bin/awk
```

It's currently fine but you might want to know that there is a `lib.getExe` which essentially gives the same output by relying on `meta.mainProgram` or throwing a warning and guessing using the pname. Note that there is also a `lib.getExe'` where you provide the binary name as string. 
```suggestion
            --replace /usr/bin/awk ${lib.getExe gawk}
```

---

Fixed